src/main/resources/edu/cornell/kfs/fp/document/datadictionary/RecurringDisbursementVoucherDocument.xml

```diff
@@ -130,6 +131,31 @@
 		</property>
 	</bean>
 
+	<bean id="RecurringDisbursementVoucherDocument-validations" parent="RecurringDisbursementVoucherDocument-validations-parentBean" />
+	<bean id="RecurringDisbursementVoucherDocument-validations-parentBean" abstract="true" 
+		parent="DisbursementVoucherDocument-validations-parentBean" class="org.springframework.beans.factory.config.MapFactoryBean">
```

Comment:
I see that this bean is a child of a base financials parent bean instead of a base financials non-parent bean. Was that necessary to avoid factory-bean-related issues? Or are there customizations for regular DVs that you needed to exclude from the recurring DVs?

Also, do you need to specify the class again if the parent bean already defines it?

Reply:
I should be able to switch to non parent, and I will remove the class definition